web-registry/src/lib/ledger.ts

```diff
@@ -8,8 +8,8 @@ export const expLedger = process.env.REACT_APP_EXP_LEDGER;
 const expUriBase = 'exp-';
 const uriBase = `${getApiUri()}/${expLedger ? expUriBase : ''}`;
 
-// Simple proxy endpoint for REST requests. We check for chainId as an on/off switch.
-export const ledgerRESTUri = chainId ? `${uriBase}ledger-rest` : undefined;
```

Comment:
@blushi I removed this check because there are cases (like the new credit class page) where ledger-rest requests are not experimental, as I mentioned in slack. https://regen-network.slack.com/archives/C01LX9E8QN8/p1648155852710709?thread_ts=1647993852.056959&cid=C01LX9E8QN8

Reply:
sounds good to me, although "experimental" is not the right word here (since experimental is meant to refer to features on hambach)